File: cses/introductoryProblems/missingNumber.java
import java.util.Scanner;

public class missingNumber
{
	public static void main(String[] args)
	{
		Scanner sc = new Scanner(System.in);
		long n = sc.nextLong();
		long sum = n * (n + 1)/2;
		for (long i = 0; i < (n - 1); i++)
		{
			long num = sc.nextLong();
			sum = sum - num;
		}
		System.out.println(sum);
		sc.close();
	}
}

File: cses/introductoryProblems/missingNumber.cpp
#include <iostream>
using namespace std;

int main(void)
{
	long n;
	cin >> n;
	long sum = n * (n + 1)/2;
	for (int i = 0; i < (n - 1); i++)
	{
		long num;
		cin >> num;
		sum = sum - num;
	}
	cout << sum << endl;
	return 0;
}
